app/build.gradle

```diff
@@ -30,4 +30,5 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+    implementation 'com.android.support:recyclerview-v7:28.0.0'
```

Comment:
could there be any issues with hardcoding version numbers into the actual `implementation`?

Reply:
Absolutely, I'm glad you asked, potential future work-mate! Hardcoding version numbers is especially a bad idea if you have multiple dependencies with the same version number, that have to remain in sync. In that case, you should always use a variable, as I did in my follow-up commit :)